app/models/group.js

```diff
@@ -59,12 +59,13 @@ groupSchema.pre('save', function (next) {
   if (this.inLDAP) {
     return next();
   }
+  var that = this;
```

Comment:
It's not enough to stop me from merging it -- I'll probably fix it at some point but it's not an issue for now...just wanted to have it on record :smile_cat: